File: Nursery/App/HomeDefaults.swift
import Foundation

/// The developer's home setup: his go2rtc server, its Tailscale address, and his stream names.
/// They are pre-filled, so his phones need no setup.
/// A public build sets them to empty strings. Nothing else must be deleted.
/// The addresses and the stream names are used only as the initial values of `Settings` and as
/// a placeholder in the settings. A value that the user saved always wins.
/// `configPath` is read at each config load. When it is empty, the app loads no config and
/// offers no camera control.
enum HomeDefaults {
    /// The go2rtc server on the home Wi-Fi (a Raspberry Pi).
    static let serverHost = "192.168.0.136"
    /// The same server on the tailnet ("rpi-host"), for use away from home.
    static let remoteHost = "100.104.188.72"
    /// The go2rtc stream names: the camera's main (high) stream and its sub (low) stream.
    static let streamMain = "nursery"
    static let streamSmall = "nursery_sd"
    /// The file that go2rtc serves, with the Home Assistant webhook ids. See `CameraControl`.
    static let configPath = "nursery/config.js"
}

File: android/app/src/main/java/cz/drabek/chuvicka/HomeDefaults.kt
package cz.drabek.chuvicka

/**
 * The developer's home setup. It is pre-filled so that his phones need no setup.
 * A public build sets these to empty strings and deletes nothing else.
 * They are used only as the initial values of Settings and in the wizard placeholders.
 */
object HomeDefaults {
    /** The Pi with go2rtc, in the home network. */
    const val SERVER_HOST = "192.168.0.136"
    /** The Pi's Tailscale address, for the time away from home. */
    const val REMOTE_HOST = "100.104.188.72"
    /** The go2rtc stream names: the camera's main (high) stream and its sub (low) stream. */
    const val STREAM_MAIN = "nursery"
    const val STREAM_SMALL = "nursery_sd"
}

/** The go2rtc default ports. They are go2rtc's defaults, not the owner's setup. */
object Go2rtc {
    const val RTSP_PORT = 8554
    const val API_PORT = 1984
}
